src/components/WalletModal/index.tsx

```diff
@@ -336,8 +396,11 @@ export default function WalletModal({
     }
 
     return (
-        <Modal isOpen={walletModalOpen} onDismiss={toggleWalletModal} minHeight={false} maxHeight={90}>
-            <Wrapper>{getModalContent()}</Wrapper>
-        </Modal>
+        <>
+            <Modal isOpen={walletModalOpen} onDismiss={toggleWalletModal} minHeight={false} maxHeight={90}>
+                <Wrapper>{getModalContent()}</Wrapper>
```

Comment:
why is this a function and not a component?

Reply:
It was previously implemented by some other developers as a function, I have converted it to a functional component. Please review and approve